modules/core/src/main/java/org/apache/synapse/core/axis2/SynapseCallbackReceiver.java

```diff
@@ -181,12 +181,19 @@ public void receive(MessageContext messageCtx) throws AxisFault {
                 messageCtx.removeProperty(PassThroughConstants.INTERNAL_EXCEPTION_ORIGIN);
                 org.apache.synapse.MessageContext SynapseOutMsgCtx = callback.getSynapseOutMsgCtx();
                 ConcurrencyThrottlingUtils.decrementConcurrencyThrottleAccessController(SynapseOutMsgCtx);
-
+                boolean dropMessage = false;
```

Comment:
can we please change this dropMessage variable name to a more meaningful one?

Reply:
renamed to isMarkedForRemoval